importer/helpers/dagbuilder.go

```diff
@@ -33,6 +35,9 @@ type DagBuilderParams struct {
 	// instead of using the unixfs TRaw type
 	RawLeaves bool
 
+	// CID Version to use
+	CidVersion dag.CidVersion
```

Comment:
I think instead of using a struct for this, we should just use a `cid.Prefix`. That way we make it much easier for us to switch the hash function in the future